mindone/transformers/mindspore_adapter/train_onestep_wrapper.py

```diff
@@ -139,8 +140,23 @@ def __init__(
             reducer = create_grad_reducer(network.trainable_params())
             is_zero = False
 
+        # scaler and reducer
+        assert "ms_loss_scaler" not in scaler_config
+        if scaler.lower() in ("default", "static"):
+            _scaler_config = {"scale_value": 1024}
+            _scaler_config.update(scaler_config)
+            scaler = create_loss_scaler("static", **_scaler_config)
+        elif scaler.lower() in ("auto", "dynamic"):
+            scaler = create_loss_scaler("dynamic", **scaler_config)
+        elif scaler.lower() == "none":
+            scaler = create_loss_scaler("none", **scaler_config)
+        else:
+            raise NotImplementedError
```

Comment:
Please specify `scaler` possible values with `Literal` in the `__init__()` definition.